taichi/codegen/codegen_llvm.h

```diff
@@ -896,8 +896,18 @@ class CodeGenLLVM : public IRVisitor, public ModuleBuilder {
   }
 
   void visit(AssertStmt *stmt) override {
-    stmt->value = call("taichi_assert", get_context(), stmt->val->value,
-                       builder->CreateGlobalStringPtr(stmt->text));
+    Value *text;
+    if (stmt->args.empty()) {
+      text = builder->CreateGlobalStringPtr(stmt->text);
+    } else {
+      std::vector<Value *> args;
+      for (auto arg : stmt->args) {
```

Comment:
Maybe we need `text` here in `args`?

Reply:
Good catch! (but still doesn't work now)